var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -336,6 +345,15 @@ def die_without_fortran(self):
                 'OpenMPI requires both C and Fortran compilers!'
             )
 
+    @property
+    def force_autoreconf(self):
+        return self.spec.satisfies('@develop')
```

Comment:
Is this actually needed? This attribute simply causes the deletion of the `configure` script, as the default `autoreconf` method is skipped if `configure` already exists.